planning/obstacle_avoidance_planner/src/utils/utils.cpp

```diff
@@ -708,6 +711,35 @@ bool isOutsideDrivableAreaFromRectangleFootprint(
     tier4_autoware_utils::calcOffsetPose(traj_point.pose, -base_to_rear, -base_to_left, 0.0)
       .position;
 
+  LinearRing2d footprint_polygon;
```

Comment:
https://github.com/autowarefoundation/autoware.universe/blob/3c43118a876d58bba4949d876302ff804436f37f/planning/obstacle_avoidance_planner/src/utils/utils.cpp#L743-L774

This implementation is the original one not considering the edge lines of footprints.
With this PR, this function is almost disabled.

Could you add a option to choose the method from yours and the original one.

Reply:
What is the reason not to consider edge lines of footprints? I mean why this wouldn't be wanted?

Reply:
Oh I see. Then makes sense for now to have an option to enable this. Thank you.

@beyzanurkaya maybe we can add a param like `is_considering_footprint_edges` or `do_consider_footprint_edges` under [option](https://github.com/autowarefoundation/autoware_launch/blob/1237386ebc92eacaa2327834eb06bf88f03eee28/autoware_launch/config/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/obstacle_avoidance_planner.param.yaml#L3)

Reply:
@takayuki5168 I added a flag for this PR.